tests/php/tests/includes/helper/test_class.wp-job-manager-com-auth-token.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+require_once JOB_MANAGER_PLUGIN_DIR . '/includes/helper/class-wp-job-manager-com-auth-token.php';
+
+/**
+ * @group helper
+ * @group helper-base
+ */
+class WP_Test_WP_Job_Manager_Com_Auth_Token extends WPJM_BaseTest {
```

Comment:
I'm not sure what the best way to test expired tokens here..if anyone has any suggestions, please feel free to mention it. :)

Reply:
Yeah, I implemented that this morning. Thanks for the feedback!